src/Team.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react';
+import teamMember from './components/assets/image/teamMember.jpg';
+import karen from './components/assets/image/karen.jfif';
+import Navbar from './components/Navbar';
```

Comment:
Codacy found an issue: ['Navbar' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=6047701)